driver/ppm_fillers.c

```diff
@@ -195,24 +195,23 @@ int f_sys_single_x(struct event_filler_arguments *args)
 	return add_sentinel(args);
 }
 
-static inline uint32_t get_fd_dev(int64_t fd)
+static inline bool get_fd_dev_ino(int64_t fd, uint32_t* dev, uint64_t* ino)
```

Comment:
Also here we can do like in BPF. We can initialize `dev` and `ino` at the beginning of the caller function and then `get_fd_dev_ino` could become a `void` function without returning any `bool` like in BPF.